src/vg/civcraft/mc/civchat2/CivChat2Manager.java

```diff
@@ -296,11 +299,12 @@ public void broadcastMessage(Player sender, String chatMessage, Set<Player> reci
 						//reciever is in differnt world dont send
 						continue;
 					} else {
-						receiver.sendMessage(sb.append(color) 
+						receiver.sendMessage(String.format(event.getFormat(), sender.getName(), chatMessage));
```

Comment:
Given this is a civ-series plugin, go ahead and instead of sender.getName() use NameAPI.getCurrentName(sender.getUniqueId())